smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -520,14 +520,17 @@ def poll() {
                 return "Poll completed $map - $state.hubs"
             }
         } catch (groovyx.net.http.HttpResponseException e) {
-            if (e.statusCode == 401) { // token is expired
-                state.remove("HarmonyAccessToken")
-                return "Harmony Access token has expired"
-            }
-		} catch(Exception e) {
-        	log.trace e
-		}
-	}
+              if (e.statusCode == 401) { // token is expired
+                  state.remove("HarmonyAccessToken")
+                  log.warn "Harmony Access token has expired"
+              }
+        } catch (java.net.SocketTimeoutException e) {
+        	log.warn "Connection to the hub timed out. Please restart the hub and try again."
+              state.resethub = true
+        } catch (e) {
+        	log.warn "Hostname in certificate didn't match. Please try again later."
```

Comment:
Is this log statement accurate?  Because in this case e could be any type of exception that hasn't already been caught, which could make debugging ambiguous if you his this block